Calendar: add the showWeekNumbers property description (#4288)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxCalendar/1 Configuration/showWeekNumbers.md

---
---
##### shortDescription

> Specifies whether or not the UI component displays a column with the week numbers.

Reviewer: ```suggestion
Specifies whether to display a column with week numbers.
```

---

> The [weekNumberRule](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#weekNumberRule) property defines the beginning of the first week of year.

Reviewer: `weekNumberRule` has its own description. Let's just mention this property in the See Also section.
